### PR TITLE
Add Panther Landing Page

### LINKED ISSUE #340: App Homepage
### Describe the ideal solution

Today, the first page a user will visits is the Cloud Security page. In order to better guide their usage of the product, the app should have a general Home page to provide a quick explanation of each major component (cloud security, log analysis, historical) and a way to click through to each section. 

### Describe your use cases

The use case is primarily a generalized home page for the app. 

### How frequently would you use such feature

Daily

### Describe alternatives you have considered

Nothing

### References

N/A

### Additional context

N/A

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

This PR adds a basic landing page that the users will initially see when logging in to panther. It's the default page that both new  & recurring users will visit.

Closes #340 

## Changes

- Add route for landing page (prevent redirect to compliance overview)
- Add Landing page

## Testing

- Locally

## Screenshots

Screen Shot 2020-03-24 at 12 17 05 AM
Screen Shot 2020-03-24 at 12 17 12 AM

### CODE REVIEW

File: web/src/pages/Landing/Landing.tsx [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="user" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Invite your team
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Create multiple users and get your team onboarded to Panther
+              </Text>
+              <Text color="blue300" p={4} is={Link} to={urls.settings.users()} size="large">
+                Create your Team
```
Comment (jacknagz) [inline web/src/pages/Landing/Landing.tsx]: Manage Users

File: web/src/pages/Landing/Landing.tsx [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+              </Text>
+            </Flex>
+            <Flex flexDirection="column" justifyContent="center" alignItems="center" px={10} py={5}>
+              <Icon color="grey300" type="infra-source" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Setup Infrastructure Monitoring
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Connect AWS accounts to monitor their compliance to your certain policies
```
Comment (jacknagz) [inline web/src/pages/Landing/Landing.tsx]: Connect AWS accounts scan resources and detect misconfigurations

File: web/src/pages/Landing/Landing.tsx [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+                Connect your log buckets in order to allow Panther to run rules against them
+              </Text>
+              <Text
+                color="blue300"
+                p={4}
+                is={Link}
+                to={urls.logAnalysis.sources.create()}
+                size="large"
+              >
+                Connect log bucket
```
Comment (jacknagz) [inline web/src/pages/Landing/Landing.tsx]: Connect S3 Buckets

File: web/src/pages/Landing/Landing.tsx [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+                Onboard an AWS account
+              </Text>
+            </Flex>
+            <Flex flexDirection="column" justifyContent="center" alignItems="center" px={10} py={5}>
+              <Icon color="grey300" type="log-source" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Setup your Log Sources
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Connect your log buckets in order to allow Panther to run rules against them
```
Comment (jacknagz) [inline web/src/pages/Landing/Landing.tsx]: Connect your log buckets and analyze data with rules

File: web/src/pages/Landing/Landing.tsx [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+                Launch Athena
+              </Text>
+            </Flex>
+          </Grid>
+        </Card>
+      </Box>
+      <Box borderTop="1px solid" borderColor="grey100" my={60}>
+        <Box is="header" my={10}>
+          <Heading is="h1" size="large" textAlign="center" color="grey500" mb={4}>
+            The following links may seem helpful
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `The following links may be helpful` is a better phrasing I think.

File: web/src/pages/Landing/Landing.tsx [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+        </Box>
+        <Card mb={6} is="section">
+          <Grid gridTemplateColumns="repeat(3, 1fr)" py={5}>
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="user" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Invite your team
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Create multiple users and get your team onboarded to Panther
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `Create users and get your team onboarded to Panther`

File: web/src/pages/Landing/Landing.tsx (hunk 2 above)
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `Connect AWS accounts to monitor them with Cloud Security policies`

File: web/src/pages/Landing/Landing.tsx (hunk 4 above)
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `Connect your log sources to evaluate logs with Log Analysis rules`

File: web/src/pages/Landing/Landing.tsx [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+
+        <Card mb={6} is="section">
+          <Grid gridTemplateColumns="repeat(3, 1fr)" py={5}>
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="output" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Setup an Alert Destination
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Add notification channels for suspicious activity or misconfiguration detection
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `Add destinations so Panther can notify you of policy and rule findings`

File: web/src/pages/Landing/Landing.tsx [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+              </Text>
+            </Flex>
+
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="policy" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Write Infrastructure Policies
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Create policies that your AWS infrastructure must abide to
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `Create Cloud Security policies to evaluate your AWS infrastructure`

File: web/src/pages/Landing/Landing.tsx [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+                Create a Policy
+              </Text>
+            </Flex>
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="rule" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Write Log Detection Rules
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Create rules to run against your logs and trigger alerts on suspicious activity
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `Create rules to evaluate your logs and trigger alerts on suspicious activity`

File: web/src/pages/Landing/Landing.tsx [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+        </Card>
+        <Card mb={6} is="section">
+          <Grid gridTemplateColumns="repeat(3, 1fr)" py={5}>
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="infra-analytics" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Triage Alerts
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                View alerts about suspicious activity according to your Rules
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `View alerts generated by Rules`

File: web/src/pages/Landing/Landing.tsx [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+                View Alerts
+              </Text>
+            </Flex>
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="infra-analytics" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Search through Resources
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                View your connected AWS resources and monitor their health
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `View your AWS resources and monitor their compliance with your Cloud Security policies`

File: web/src/pages/Landing/Landing.tsx [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+                View Resources
+              </Text>
+            </Flex>
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" px={10} py={5}>
+              <Icon color="grey300" type="infra-analytics" mb={4} size="large" />
+              <Text size="large" is="h4" color="grey500" mb={4}>
+                Query Logs with Athena
+              </Text>
+              <Text size="medium" is="p" color="grey300" textAlign="center" maxWidth={250}>
+                Use AWS Athena to write complex queries against formatted logs
```
Comment (nhakmiller) [inline web/src/pages/Landing/Landing.tsx]: `Use AWS Athena to write complex queries against your normalized log data`

**Comment by kostaspap** [comment]
Damn, I like this change!

**Pull request merged**